addon/components/frost-multi-select.js

```diff
@@ -58,9 +57,9 @@ export default FrostSelect.extend({
   },
 
   getValues () {
-    const selected = this.get('selectedItems')
+    const selected = this.get('selectedItems') || []
```

Comment:
We may want to get in the habit of wrapping things like `[]` in `Ember.A([])` in case we (or someone else) has prototype extensions disabled - which would make operations like `map` unavailable to the vanilla `[]`

Just a note, nothing urgent